api/src/main/java/org/openmrs/Order.java

```diff
@@ -807,4 +823,44 @@ public Date getEffectiveStopDate() {
 		return this.getDateStopped() != null ? this.getDateStopped() : this.getAutoExpireDate();
 	}
 	
+	/**
+	 * @since 1.12
+	 * {@link org.openmrs.OrderGroup}
+	 * @returns the OrderGroup
+	 */
+	public OrderGroup getOrderGroup() {
+		return orderGroup;
+	}
+	
+	/**
+	 * @since 1.12
+	 * Sets the OrderGroup for that order.
+	 * If the order is ordered independently, it does not set an orderGroup for it.
+	 * If the order is ordered as an orderGroup, then sets a link to the OrderGroup for that particular order.
+	 * @param orderGroup
+	 */
+	public void setOrderGroup(OrderGroup orderGroup) {
+		this.orderGroup = orderGroup;
+	}
+	
+	/**
+	 * @since 1.12
+	 * Gets the sortWeight for an order if it is ordered as an OrderGroup.
+	 * Doesn't get the sortWeight if the order is not ordered as an OrderGroup.
```

Comment:
Should include text like:

> \* \<tt>sortWeight\</tt> is used internally by the API to manage the sequencing of orders when grouped.
> \* This value may be changed by the API as needed to manage order group sequencing. Clients should
> \*  rarely (if ever) need to use this property and, instead, we recommend inferring the sequence from the
> \* \<tt>List\</tt> of orders obtained from the group.

Also, the comment says "Doesn't get the sortWeight if the order is not ordered as an OrderGroup," the method does nothing to enforce this behavior (returns sortWeight regardless of whether the order is in a group).